dashboard/v1.1/src/pages/Monitoring/Matrix.tsx

```diff
@@ -137,7 +137,12 @@ const Element: FC<ElementProps> = ({
   return (
     <TableRow>
       <TableCell
-        style={{ maxWidth: 240, fontSize: 16, overflowX: 'hidden' }}
+        style={{
```

Comment:
@ruddi10 I don't feel this is the best approach to fix this bug. If you check minutely, after the changes you would find the table broken. Instead, you can use the `substr()` to trim the content [here](https://github.com/bench-routes/bench-routes/blob/master/dashboard/v1.1/src/pages/Monitoring/Matrix.tsx#L104). You can trim the sentence to something nearly 50 or 60 characters, whichever seems good in the table area.

Reply:
@muskankhedia yup missed that will trim the route then

Reply:
Yes, you have to trim it. But keep in mind not to change the tooltip related to it.